Source/WebCore/platform/graphics/mac/GraphicsChecksMac.cpp

```diff
@@ -65,9 +70,7 @@ static io_connect_t attachToAppleGraphicsControl()
         return IO_OBJECT_NULL;
 
     io_connect_t dataPort;
-    IOObjectRetain(serviceObject);
```

Comment:
I'm not confident it is safe to remove the Retain/Release pair. Can you explain why this is safe?

Reply:
Reference count goes up by one and down by one. 

The tools folder has a code snippet like this without the retain and release. 

If the retain was needed, the release should not be there.

Reply:
I did some looking, and do not see other places where a Retain/Release is needed around IOServiceOpen, so this change is probably good.